src/cmd/server/start.ts

```diff
@@ -95,7 +95,7 @@ async function getServerOptions (
 
   function getIpcOptions () {
     return {
-      path: path.normalize(fsPath) + path.sep + 'socket',
+      path: path.normalize(connectionInfoDir) + path.sep + 'socket',
```

Comment:
one question here: why not using `path.join` instead of `+ path.sep +`?

Reply:
I don't remember any reason, so I'll change it